index.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable brace-style, camelcase, semi */
+/* global R5 */
+
+module.exports = Mailer;
+
+if (!global.R5) {
+  global.R5 = {
+    out: console
+  }
+}
+
+let mailer = require('mailgun-js');
+let config = {
+  domain:  'funnode.com',
+  key: '131133',
```

Comment:
what is this key?? this code won't work at all without the correct domain and key values provided by the user